dev/build-arrow-ballista.sh

```diff
@@ -24,7 +24,7 @@ rm -rf arrow-ballista 2>/dev/null
 
 # clone the repo
 # TODO make repo/branch configurable
-git clone https://github.com/apache/arrow-ballista
+git clone --branch join_filter https://github.com/korowa/arrow-ballista
```

Comment:
I think this change probably needs to be reverted prior to merge

Reply:
I guess it should be merged as it is, then. we update datafusion revision in related ballista PR, and after ballista PR is merged we reset this back to ballista master

Reply:
Sounds good. The corresponding ballista PR seems to be https://github.com/apache/arrow-ballista/pull/36 👍